### ISSUE TITLE
How to save data on table?

### ISSUE THREAD

**Issue by shirshak55** [64robots/nova-row#7]
it always show [object Object],[object Object] so how should I persist the data in database. I have made price column with json field type.

Here is field detail.

```

            Row::make('Price', [
                Text::make('Currency')
                  ->fieldClasses('w-full px-8 py-6')
                  ->hideLabelInForms(),

                Number::make('Amount')
                  ->fieldClasses('w-full px-8 py-6')
                  ->hideLabelInForms(),

              ])->fieldClasses('w-full px-8 py-6')
                ->labelClasses('w-1/2 px-8 py-6'),
```

Now when i submit post in database it is stored as [object Object] etc..

**Comment by dillingham** [comment]
https://github.com/64robots/nova-row/issues/3

**Comment by disitec** [comment]
This has been fixed on 1.0.1. Now is stored a JSON string by default. 

You always can override this behavior doing the following:

```php
Row::make('Content', [
                // your fields

              ])->fillUsing(function($request, $model, $attribute, $requestAttribute) {
            $decoded_payload = json_decode($request[$requestAttribute]);
            // your logic
            $model->{$attribute} = $whatever;
        });
```
